website/docs/r/policy_predefined_gateway_policy.html.markdown

```diff
@@ -0,0 +1,101 @@
+---
+subcategory: "Policy - Firewall"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_predefined_gateway_policy"
+description: A resource to update Predefined Gateway Security Policies.
+---
+
+# nsxt_policy_predefined_gateway_policy
+
+This resource provides a method to fine-tune a pre-created Gateway Policy and its rules.
+There are two separate use cases for this resource:
+* Modify certain settings of default Gateway Policy and its Default Rule.
+* Modify predefined Gateway Policy that is not listed under Default category, and add rules to it.
+  This use case is relevant for VMC.
+
+This resource is applicable to NSX Global Manager, NSX Policy Manager and VMC.
```

Comment:
Actually curious what is the official term for Policy? Is it "Policy" or "Policy Manager"?

Reply:
I think Policy Manager..